CHANGELOG_PENDING.md

```diff
@@ -1,4 +1,5 @@
 ### SDK Features
+* SDK generate errors are fixed to use pointer receivers preventing confussion, and potential impossible type assertions. the SDK will only return API generated API error types as pointers. This fix ensures Go's type system will catch invalid error type assertions.
```

Comment:
Nit pick:
generate -> generated 
confussion -> confusion
potential impossible type assertions. `the` SDK will -> potential impossible type assertions. `The` SDK will